pkg/client/v1/nimbleos/access_control_record.go

```diff
@@ -76,3 +94,27 @@ type AccessControlRecord struct {
 	// AccessProtocol - Access protocol of the volume.
 	AccessProtocol *NsAccessProtocol `json:"access_protocol,omitempty"`
 }
+
+// Struct for AccessControlRecordFields
+type AccessControlRecordStringFields struct {
```

Comment:
I don't love this name, but I failed to come up with something better for this peculiar concept.  Maybe AccessControlRecordFieldHandles?

Reply:
Yeah, it took me a while to come up with a name as well. 
Tagging the others to join the conversation -- @raunakkumar @rgcostea @suneeth51 @vidhutsingh

Reply:
i don't have a better suggestion too. `AccessControlRecordFieldHandles`  works for me